src/tools/TSDMain.java

```diff
@@ -13,10 +13,17 @@
 package net.opentsdb.tools;
 
 import java.io.IOException;
+import java.lang.reflect.Constructor;
```

Comment:
I probably don't need all of these anymore, I will go back and see which ones are actually needed.